TwinePlatform/twine-monolith
# ♻️ Sort month labels chronologically

Changes made as part of adding projects chart (see #335)

### Changes
- Fix type `abreviated` -> `abbreviated`
- Add `sortFormatted` function to sort formatted date strings chronologically
- Add `Strings.truncate` function
- Truncate labels at 10 chars if they are not date strings

### Testing Requirements
  - Check dashboard landing page statistics make sense and are formatted right
  - Check order of months is chronological on all charts

### Release Requirements
- Not yet.

### Manual Deployment
- N/A
